setup.py

```diff
@@ -102,19 +102,31 @@ def collect_external_package_data(external_package_data):
     include_dirs.append(f"{EIGEN}")
 
 else:
-    # Opencv
-    opencv_include_dirs = [
-        "/usr/local/opt/opencv/include",  # old opencv brew (v3)
-        "/usr/local/opt/opencv@3/include",  # new opencv@3 brew
-        "/usr/local/include/opencv4",  # new opencv brew (v4)
-        "/usr/include/opencv4",  # opencv (v4) on ubuntu 20.04
-    ]
-    opencv_library_dirs = [
-        "/usr/local/opt/opencv/lib",  # old opencv brew (v3)
-        "/usr/local/opt/opencv@3/lib",  # new opencv@3 brew
-        "/usr/local/lib",  # new opencv brew (v4)
-        "/usr/lib",  # opencv (v4) on ubuntu 20.04
-    ]
+    # Check for anaconda / miniconda install
+    is_conda = 'CONDA_PREFIX' in os.environ
```

Comment:
In addition to checking if the variable is present, one should also check if it is not set to an empty string.
```suggestion
    is_conda = os.environ.get('CONDA_PREFIX', "") != ""
```